src/main/java/io/airlift/junit/InitializeLogging.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.airlift.junit;
+
+import io.airlift.log.Level;
+import io.airlift.log.Logging;
+import org.junit.jupiter.api.extension.Extension;
+
+public class InitializeLogging
+        implements Extension
+{
+    static {
+        Logging logging = Logging.initialize();
+
+        // disable logging from testcontainers by default
+        logging.setLevel("tc", Level.OFF);
```

Comment:
Remove this. It’s useful to see when containers are pulled, starting, how long they took, etc.

Reply:
Sure, I can remove it for now. However, I find the logs from TestContainers  to be very verbose. Combined with everything else, it's the proverbial "death by a thousand cuts" that makes it impossible to find what caused a test failure in the sea of logs.